lib/tizen_doctor.dart

```diff
@@ -112,16 +108,22 @@ class TizenValidator extends DoctorValidator {
       return ValidationResult(ValidationType.partial, messages);
     }
 
-    if (dotnetCli == null) {
+    if (globals.processManager.canRun(dotnetCli.path)) {
```

Comment:
To remove exception message when dotnet cli is not installed on the system.

```suggestion
    if (dotnetCli != null && globals.processManager.canRun(dotnetCli.path)) {
```

Reply:
Ooops. Thanks.